app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx

```diff
@@ -0,0 +1,30 @@
+"use client";
+import { motion } from "framer-motion";
+import { Typography } from "@/components/ui/typography";
+import Link from "next/link";
+
+export default function GitBasics() {
+    return (
+        <motion.div
+            initial={{ opacity: 0, y: 20 }}
+            animate={{ opacity: 1, y: 0 }}
+            exit={{ opacity: 0, y: -20 }}
+            transition={{ duration: 0.5 }}
+            className="container mx-auto px-4 py-8"
+        >
+            <Typography variant="h1" className="mb-8 text-4xl border-b border-red-500 pb-2">
+                Bitcoin Basics
+            </Typography>
+            <Typography variant="body" className="mb-4">
+                Bitcoin is kind of a strange thing for people to grasp, mostly because you can’t physically grasp it, but it can still be used to purchase goods and services.
+
```

Comment:
Newlines and whitespace are not respected by HTML tags. Use separate components or a break tag <br/> for spacing.

Reply:
added <br></br> to lines, thanks